Fauna-RPG/Buffs/Buff_Poison.gd

```diff
@@ -0,0 +1,13 @@
+extends BuffBase
+class_name Buff_Poison
```

Comment:
Good point, we may want to differentiate between them so we can apply in the correct order. Maybe it's better to have a bool like `is_debuff` instead, since they're pretty much the same thing

Reply:
Having a class will allow for type check, I think it'll be clearer also

Reply:
That's true. I think the parent Status should do pretty much everything, but we can use the child Buff/Debuff for easy initialization of visual stuff. For example, Buff could have a green border in the UI, while Debuff has red. Might also be useful for ordering later on like you mentioned